src/test/java/uk/gov/companieshouse/web/accounts/transformer/smallfull/FixedAssetsTransformerImplTests.java

```diff
@@ -4,10 +4,7 @@
 import org.junit.jupiter.api.Test;
 import uk.gov.companieshouse.api.model.accounts.smallfull.BalanceSheetApi;
 import uk.gov.companieshouse.api.model.accounts.smallfull.FixedAssetsApi;
-import uk.gov.companieshouse.web.accounts.model.smallfull.BalanceSheet;
-import uk.gov.companieshouse.web.accounts.model.smallfull.FixedAssets;
-import uk.gov.companieshouse.web.accounts.model.smallfull.FixedInvestments;
-import uk.gov.companieshouse.web.accounts.model.smallfull.TangibleAssets;
+import uk.gov.companieshouse.web.accounts.model.smallfull.*;
```

Comment:
Can we revert to use explicit imports please?

Reply:
I fixed that :(
no idea why it still shows up